f2/src/components/button/index.js

```diff
@@ -13,7 +13,7 @@ export const Button = ({ variant, variants, variantColor, ...props }) => (
       ? {
           borderBottomWidth: '3px',
           borderBottomColor: `${variantColor}.800`,
-          bg: `${variantColor}.600`,
+          bg: `${variantColor}.700`,
```

Comment:
Not sure if this is the best way to apply the color change for the feedback button in chakra, as this will also impact all non-default buttons themes. happy to discuss this further  :)

Reply:
The only thing I can think of is that we are losing the 'active' state styles: when a button is mousedown. 

I see two options: 
1. Choose an even darker color for the active state
2. We make sure the text is larger than 18px. AAA minimum color contrast for that size is 4.5, which we are passing on the buttons at 5.65
<img width="284" alt="Screen Shot 2020-02-24 at 8 32 47 AM" src="https://user-images.githubusercontent.com/5230720/75156493-3f20d200-56e0-11ea-883e-57e115882dd9.png">

Reply:
I would maybe focus more towards the footer links that are barely passing at 4.61

Reply:
hmmm, little confused as the current 
<img width="995" alt="Screen Shot 2020-02-24 at 11 28 46 AM" src="https://user-images.githubusercontent.com/2603117/75171053-0e996200-56f9-11ea-90f9-7162694debcf.png">
footer links and color contrast pass the required checks

Reply:
will check against master now and see if we still have this issue

Reply:
@khalidelaggan They all pass because they are considered large text. I say we merge this without changing the button color.

Reply:
As this form is inherited in each page, I would rather fix this minor adjustment for the button color now and have us in a clean state (0 issues)reported by any a11y check tools instead.